vowpalwabbit/core/include/vw/core/array_parameters_sparse.h

```diff
@@ -109,7 +111,7 @@ class sparse_parameters
   void share(size_t /* length */);
 #endif
 
-private:
+public:
```

Comment:
why make these vars public? did you just want to make the function public?

Reply:
yea that was for testing, reverted it